### ISSUE TITLE
Share project block

### ISSUE THREAD

**Issue by bates64** [bleush38p/EasyExtend#1]
You may want to force Scratch to share the project by using JSshareProject:

``` javascript
var desc = {
    blocks: [
        ['B', 'share project', 'share'],
    ]
};

ext.share = function() {
    JSshareProject();
}
```

**Comment by bleush38p** [comment]
Are you suggesting a block to share the project? Or a builtin ability? I'm confused. Actually, can you use `JSshareProject()` anywhare? If it does what it sounds like it does, can I just put it in the soon-to-exist EasyExtend menu so sharing doesn't involve lots of complicated steps?

**Comment by bleush38p** [comment]
Also, sorry I haven't been keeping up much with EEXT development lately. Finals :/ ... Once the week's over I'll try to get some things done.

**Comment by bleush38p** [comment]
Closing for now... issue isn't clear and is super low priority for now.
